root/tree/index/runindex64.C

```diff
@@ -8,9 +8,11 @@ const char* fname = "index64.root";
 // Apple M1 has long double == double; these values exceed its range
 // and cannot be represented as (even temporary) expression results.
 // There would be a warning if you'd try.
-static constexpr bool shortlongdouble = sizeof(long double) < 16; // was true for __APPLE__ and __arm64__
-const Long64_t bigval   = shortlongdouble ? 0xFFFFFFFFFFFF : 0xFFFFFFFFFFFFFFF; // still positive number
+static constexpr bool shortlongdouble = sizeof(long double) < 16 || true; // was true for __APPLE__ and __arm64__
+const Long64_t bigval   = shortlongdouble ?  0xFFFFFFFFFFFF :  0xFFFFFFFFFFFFFFF; // still positive number
```

Comment:
What is this condition? How does `bigval` affect the reading and under which circumstances?

Reply:
This was here before. Apparently, TTree::Print() goes into overflow at different max-values, depending on the platform.

Reply:
See f3c97809c9064feccaed3844007de9e7c6a5980d for some explanation.